stats/cloud/collector.go

```diff
@@ -291,18 +293,18 @@ func (c *Collector) Collect(sampleContainers []stats.SampleContainer) {
 	}
```

Comment:
Not for this PR (I guess) but I think this should be in the pushMetrics part not in the `Collect` as this code is call is blocking the collecting of other outputs and metrics  ... 

@na-- another issue for 0.26.0 ?

Reply:
Not sure that's necessary - look through the code again, it's not doing anything complicated, only iterating through the metrics. All of the heavy data crunching is done elsewhere.